GoogleForCreators/web-stories-wp
# Use WordPress date format for displaying dates

## Summary

This PR sets the date format to match what the user has selected in their WordPress settings.

## Relevant Technical Choices

- Falls back to a default format if nothing is selected or provided
- Keeps the configuration to the top level view therefore the content child views do not have requirements on providers to use

## User-facing changes

The dates displayed in the List and Grid views on My Stories inside Dashboard should match the WordPress user settings.

<img width="336" alt="Screen Shot 2020-06-24 at 7 04 30 AM" src="https://user-images.githubusercontent.com/1738349/85555593-afd2ef00-b5eb-11ea-9e2f-0daa6475ed01.png">
<img width="377" alt="Screen Shot 2020-06-24 at 7 04 53 AM" src="https://user-images.githubusercontent.com/1738349/85555598-b06b8580-b5eb-11ea-9f58-9280956dd1c1.png">


## Testing Instructions

1. Visit `wp-admin/options-general.php` and change the date formatting
2. Hit save
3. Refresh the My Stories Dashboard View and see the new date formatting

---



#2334
